webapp/src/reducer.js

```diff
@@ -113,6 +114,15 @@ const outIssues = (state = [], action) => {
     }
 };
 
+const allIssues = (state = {}, action) => {
```

Comment:
Let's handle the undefined state issue here instead of elsewhere

```suggestion
const allIssuesInitialState = {
    my: [],
    in: [],
    out: [],
};

const allIssues = (state = allIssuesInitialState, action) => {
```